src/io/flutter/run/bazel/BazelFields.java

```diff
@@ -269,6 +274,12 @@ GeneralCommandLine getLaunchCommand(@NotNull Project project,
       commandLine.addParameter(additionalArgsTokenizer.nextToken());
     }
 
+    final String disableBazelHotRestartParam = "--no-enable-google3-hot-reload";
```

Comment:
lets actually swap this around and add a new 
--enable-google3-hot-restart param to the bazel integration.

That way we can change the default in bazel to use the more robust external work flows. Otherwise, only IDE users will benefit from the more robust experience by default.